lib/brcobranca/boleto/banco_brasil.rb

```diff
@@ -71,7 +71,7 @@ def conta_corrente=(valor)
       # Dígito verificador da conta corrente
       # @return [String] 1 caracteres numéricos.
       def conta_corrente_dv
-        conta_corrente.modulo11_9to2_10_como_x
+        conta_corrente.modulo11(mapeamento: {10 => 'X'})
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.